config/executor/config.yaml

```diff
@@ -19,6 +19,7 @@ task:
   utilisationEventReportingInterval: 5m
 apiConnection:
   armadaUrl: "server:50051"
+  forceNoTls: false
```

Comment:
I am assuming this will not break anything?

It is required to allow the env var to work I think, it can only replace, not create.